### ISSUE TITLE
Response Headers

### ISSUE THREAD

**Issue by gil0mendes** [vuejs/vue-resource#215]
Hey,

How can I have access to the response headers?

**Comment by gil0mendes** [comment]
The `res.headers()`, on the interceptors.response, only returns this: `Object {content-type: "application/json", cache-control: "no-cache", "": ""}` :(

**Comment by steffans** [comment]
Just pass the name to get just one header, like `res.header('content-type')`

**Comment by gil0mendes** [comment]
The server returns an Header named `Authorization`, but when I call `res.header('Authorization')` theres nothing there.

The `res.header` has access to all returns Headers?

**Comment by steffans** [comment]
Yes, without the name argument it returns all recieved headers.

**Comment by gil0mendes** [comment]
Not in my case :(

### VueResource Response

```
{content-type: "application/json", cache-control: "no-cache", "": ""}
```

### Received Headers

![screen shot 2016-02-05 at 11 19 57](https://cloud.githubusercontent.com/assets/860364/12844955/7b843d92-cbfa-11e5-8aef-693da4556411.png)

**Comment by easteregg** [comment]
I guess we need a way to access exposed headers. I encountered this problem too, but I had to finish the project, tried something totally different. Still looking for getting the headers tho. It would be great to access them. 
@gil0mendes check out this link on the matter. I opened a topic over there. 
http://stackoverflow.com/questions/34852118/vue-resource-headers-from-the-cross-origin-response-is-not-fully-available
